pkg/profiler/utils.go

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) 2022 The Parca Authors
```

Comment:
I think we should move this under `pkg` to a metadata package. We have a `pkg/discovery` which is similar. 

In the same fashion, we have packages for parsing `kallsym` and process and perf maps.